src/app/calculator/furnaces/flue-gas/flue-gas-help/flue-gas-help.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Settings } from '../../../../shared/models/settings';
+import { FlueGasService } from '../flue-gas.service';
+
+@Component({
+  selector: 'app-flue-gas-help',
+  templateUrl: './flue-gas-help.component.html',
+  styleUrls: ['./flue-gas-help.component.css']
+})
+export class FlueGasHelpComponent implements OnInit {
+  @Input()
+  currentField: string;
```

Comment:
Don't need current field as an input and getting it's value from the behavior subject subscription.

Use the subscription and remove references to currentField in the parent component where it is unused